src/main/java/qna/domain/Answer.java

```diff
@@ -1,17 +1,26 @@
 package qna.domain;
 
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
 import qna.NotFoundException;
 import qna.UnAuthorizedException;
 
-import java.util.Objects;
+@Entity
+public class Answer extends BaseEntity {
 
-public class Answer {
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
한 줄에 하나의 작업만 처리하는게 가독성을 올려줍니다.
```suggestion
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
```